assets/js/common/AvailableSoftwareUpdates/Indicator.jsx

```diff
@@ -1,13 +1,21 @@
 import React from 'react';
 import classNames from 'classnames';
 import {
-  // EOS_KEYBOARD_ARROW_RIGHT_FILLED,
+  EOS_KEYBOARD_ARROW_RIGHT_FILLED,
   EOS_ERROR_OUTLINED,
 } from 'eos-icons-react';
 
 import Tooltip from '@common/Tooltip';
 
-function Indicator({ title, critical, tooltip, icon, loading, children }) {
+function Indicator({
+  title,
+  critical,
+  tooltip,
+  icon,
+  loading,
+  navigate,
```

Comment:
Don't have strong opinions about this, but how about using `onClick` instead of `navigate`?

Reply:
You're right, I'll change it

Reply:
Isn't this specifically `navigate` though?

As later you would call it like:

```jsx
      <div
        role="button"
        tabIndex={0}
        className={classNames(
          'flex flex-row items-center border border-gray-200 p-2 rounded-md grow',
          { 'cursor-pointer': !unknown }
        )}
        onClick={onClick}
        onKeyDown={({ code }) => {
          if (code === 'Enter') {
            onClick();
          }
        }}
      >
```

???

Reply:
@jamie-suse would you be fine with onNavigate? I'd be fine with onClick but maybe onNavigate is more "neutral" 😄